specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimbackends.json

```diff
@@ -225,7 +230,7 @@
             "in": "body",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/BackendUpdateParameters"
+              "$ref": "#/definitions/BackendUpdateContract"
```

Comment:
I'd recommend naming the properties model (`BackendUpdateParameters`) as something else and naming `BackendUpdateContract` as `BackendUpdateParameters` for better consistency

Reply:
fixed

---
In reply to: [147817992](https://github.com/Azure/azure-rest-api-specs/pull/1945#discussion_r147817992) [](ancestors = 147817992)